### DISCUSSION TITLE
Citing with zettlr

### DISCUSSION THREAD

**Discussion by WombatBeutler** [Zettlr/Zettlr#4297] [Q&A]
Dear all,
I use Zettlr and Zotero for academic work. Citing in Zotero used to work fine until today: 
When I try to cite in Zettlr and type [@]  the list of reference still appears, but it has no more then 50 titles. Because there are more than 90 references in my Zotero Libary the list appearing in zettlr (which is alphabetical sorted) ends with a citation key starting with M. All references beginning with  o unitl z are not displayed. If I add a new titel with an author whos name starts for example with a B I will find it in the list, when I try to citate in Zettlr- but the title with the citationkey beginning with M will dispear from the list. It seems to me that it is not possible to display more than 50 titles. I restartet Zettlr and Zotero several times, disconntected zettlr and zotero and connect them again, but the problem remains. 
If there is anybody who has an idea what may had happen and what will help, I would by extremly thankful!!
Best Regards

**Comment by nathanlesage** [comment]
The autocomplete drop-down doesn't display all entries to be more performant. The idea is that you type a few characters of the cite key, by which the drop-down will then filter, bringing the cite keys into view that you actually need.

**Reply by WombatBeutler** [reply]
Dear nathanlesage,
thank you very much!!!
That works perfectly right :-)
best regards
